eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -55,19 +55,27 @@ parameters:
   - name: DisplayName
     type: string
     default: Validate
-
+  - name: OSName
+    type: string
+    default: ''
 
 jobs:
-  - job:
+  - job: ${{ parameters.DisplayName }}_${{ parameters.OSName }}
```

Comment:
I would put this in the displayName property instead the job id here is picky about chars in the strings and this could break.